maintenance/log/handler.go

```diff
@@ -9,23 +9,27 @@ import (
 	"strings"
 	"time"
 
+	"github.com/pace/bricks/maintenance/util"
 	"github.com/rs/zerolog/hlog"
 	"github.com/rs/zerolog/log"
 )
 
 // Handler is a middleware that handles all of the logging aspects of
 // any incoming http request
-func Handler() func(http.Handler) http.Handler {
+func Handler(ignoredPrefixes ...string) func(http.Handler) http.Handler {
```

Comment:
In order to maintain backwards compatibility, even if we in the future decide to add other options as well, please use [functional options](https://dave.cheney.net/2014/10/17/functional-options-for-friendly-apis) for this.

Reply:
This is still open.